runtime/ops/fs.rs

```diff
@@ -2018,7 +2018,13 @@ fn op_readfile_sync(
   state
     .borrow_mut::<PermissionsContainer>()
     .check_read(path, "Deno.readFileSync()")?;
-  Ok(std::fs::read(path)?.into())
+  let err_mapper = |err: Error| {
+    Error::new(
+      err.kind(),
+      format!("{}, readlink '{}'", err, path.display()),
+    )
+  };
+  Ok(std::fs::read(path).map_err(err_mapper)?.into())
```

Comment:
Maybe this should be done for op_readfile_text as well?

Reply:
Makes sense, done. Although I left the name as `readfile`, without `text` suffix, as its not important part of the IO.

Reply:
There's an issue about FS errors in general that I opened at some point. stat functions, generic open, etc.

Should those be handled as well?

Reply:
@aapoalas from what I see `stat`, `open` and almost every other method is already covered. I might reactor this whole thing out though.